### DISCUSSION TITLE
Can't see who forked my repo

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [community/community#23954] [New to GitHub]
<p>Hello! I have several repos like <a href="https://github.com/alvinseville7cf/CSharp---Exercise---Other---Laboratory-Work-4" rel="noopener nofollow ugc">this</a> one where fork number is greater than zero and nothing displayed in fork list:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/7/87664cc452271e958f6a89702dd93896a330c473.png" data-download-href="/uploads/short-url/jjNDmPKnWPAxiPMwSEgjgJyq4xR.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/42812113/181080379-b63ef6bd-e87c-4ede-b180-23d664d626b4.png" alt="image" data-base62-sha1="jjNDmPKnWPAxiPMwSEgjgJyq4xR" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/7/87664cc452271e958f6a89702dd93896a330c473_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/7/87664cc452271e958f6a89702dd93896a330c473_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/7/87664cc452271e958f6a89702dd93896a330c473_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/7/87664cc452271e958f6a89702dd93896a330c473_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 175 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Why I see nothing?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>There are a few reasons why that might not be visible, but the most common one is that the person who forked your repository was later flagged/hidden, and that can happen for any number of reasons.</p>
<p>Because it involves someone else’s account, we’re unable to be very specific.</p>

**Reply by gmoniava** [reply]
I have similar situation and when I go to the person's page who forked my repo, I can see my repo on his profile. But on my profile I can't see it, only the count.

**Comment by Synaxis** [comment]
great for copycats
